.github/workflows/test.yml

```diff
@@ -0,0 +1,23 @@
+name: Test
```

Comment:
`E2E Test` is better? 
Or put it under CI Workflow? After all, the cost of E2E is high, and e2e testing can only be done if the other tests pass

Reply:
Done, PTAL again